src/components/UpcomingQuizCard/UpcomingQuizCard.constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_DATE_FORMAT = 'dd MMM, yyyy';
```

Comment:
DEFAULT_DATE_FORMAT is hard-coded to 'dd MMM, yyyy'. Date presentation is locale-sensitive; using a fixed pattern may not be appropriate for all locales. Also date formats should be localizable or derived from the user's locale when required.

**Sugerencia:**
```typescript
Consider storing only a default pattern but document that callers can pass a locale. Alternatively, use date-fns localization APIs (format with a locale option) or expose an optional locale parameter in formatDate/useUpcomingQuizCard so UI can be localized.
```